File: lesson6/demo/src/main/java/com/springfpexample/demo/config/Dbinit.java
package com.springfpexample.demo.config;


import com.springfpexample.demo.model.GenderEnum;
import com.springfpexample.demo.model.People;
import com.springfpexample.demo.repo.PeopleRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import javax.annotation.PostConstruct;


@Service
public class Dbinit {

    @Autowired
    PeopleRepository peopleRepository;

    @PostConstruct
    public void setup(){

        People p1= new People("Ruslan","Alimbetov", GenderEnum.M,18); peopleRepository.save(p1);
        People p2= new People("VASYA","Frolov", GenderEnum.M,25); peopleRepository.save(p2);
        People p3= new People("FEDOR","Petrov", GenderEnum.M,30); peopleRepository.save(p3);
        People p4= new People("Elena","Sidorova", GenderEnum.F,40); peopleRepository.save(p4);
        People p5= new People("Asia","SINICINa", GenderEnum.F,60); peopleRepository.save(p5);
        People p6= new People("Zoya","ORLOVa", GenderEnum.F,25); peopleRepository.save(p6);


    }
}

File: lesson2/demo/demo/src/main/kotlin/com/example/demo/config/DbInit.kt
package com.example.demo.config

import com.example.demo.model.Point
import com.example.demo.repo.PointRepository
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service
import javax.annotation.PostConstruct

@Service
class DbInit {

    @Autowired
    lateinit var pointRepository: PointRepository

    @PostConstruct
    fun pointInit() {
        val point = Point(num="123",  address = "Almaty", boxNum = 30, boxBusy = 30)
        pointRepository.save(point)
        pointRepository.save(Point(num="234",  address = "Almaty", boxNum = 70, boxBusy = 30))
    }
}